src/core/MeshBVH.js

```diff
@@ -727,6 +727,7 @@ export class MeshBVH {
 		}
 
 		obb.set( otherGeometry.boundingBox.min, otherGeometry.boundingBox.max, geometryToBvh );
+		obb.matrix.copy( geometryToBvh );
```

Comment:
This line shouldn't be necessary because the above "set" operation includes the OBB matrix. However looking at the implementation it looks like there is a bug [here](https://github.com/gkjohnson/three-mesh-bvh/blob/f37651400807e106960d27287de036e655873522/src/math/OrientedBox.js#L26).

Instead of `this.matrix = matrix` it should be `this.matrix.copy( matrix )` so we don't retain the passed matrix reference. Do you mind changing that? It could potentially be causing problems. It would also be good to return "this" to align with the three.js APIs.